experimental/import-expression.md

```diff
@@ -0,0 +1,13 @@
+# [Dynamic Import Expression](https://github.com/domenic/proposal-dynamic-import)
+
+## ImportExpression
```

Comment:
It seems like "Import Expression" better follows existing ESTree constructs. "Dynamic" isn't really a syntactic description IMHO.